partials/cli-plugins/_insight-ingress-configuration.hbs.md

```diff
@@ -0,0 +1,32 @@
+<!-- Configure certificate and end point for insight cli when ingress is enabled -->
+
+To get a certificate, run:
+
+```bash
+kubectl get secret ingress-cert -n metadata-store -o json | jq -r '.data."ca.crt"' | base64 -d > insight-ca.crt
+```
+
+The endpoint host is set to `metadata-store.<ingress-domain>`,
+such as `metadata-store.example.domain.com`).
+
+This value matches the value of `ingress_domain`.
+
+If no accessible DNS record exists for the domain, edit the `/etc/hosts` file to add a local record:
```

Comment:
This part is awkward because it will work to connect to the metadata-store via the CLI, but it won't work if there is a multi-cluster setup. 

In a multi-cluster setup, there needs to be a DNS entry because the build cluster (with grype) needs to be able to access the metadata-store with the certificate and fully qualified url.

Reply:
the instructions here are specifically for setting up insight to connect to the store though, do you think we should still call out the multi-cluster setup scenario?

Reply:
I think providing these instructions means that we're assuming that there is a possible use case in which a user would install the metadata-store without a build cluster that would put things into it, and I'm not sure that is a valid use case 🤔

Reply:
made some changes, please take another look